crates/wasm/lib.rs

```diff
@@ -456,7 +452,27 @@ mod tests {
 
     let mut prev_state_bytes =
       deno_core::serde_json::to_vec(&prev_state).unwrap();
-    let state = rt.call(&mut prev_state_bytes, &mut action_bytes).unwrap();
+    let state = rt
+      .call(
+        &mut prev_state_bytes,
+        &mut action_bytes,
+        InteractionContext {
+          transaction: InteractionTx {
+            id: String::new(),
+            owner: String::new(),
+            tags: vec![],
+            target: String::new(),
+            quantity: String::new(),
+            reward: String::new(),
+          },
+          block: InteractionBlock {
+            height: 0,
+            indep_hash: String::new(),
+            timestamp: 0,
+          },
+        },
```

Comment:
This is the default implementation
```suggestion
        InterfactionContext::default()
```